fuse/2.21.0-community/health-check/booster.yaml

```diff
@@ -0,0 +1,15 @@
+name: Red Hat Fuse - Health Check Example
```

Comment:
I think this may need 
```
metadata:
￼   runsOn: '!osio'
```

￼

Reply:
How come?

Reply:
Is it tested and works in OSIO (including Che)?

Reply:
Hold on, we're discussing about this in the slack channel